includes/enrolment/class-sensei-enrolment-learner-calculation-job.php

```diff
@@ -90,9 +90,11 @@ public function run() {
 			return;
 		}
 
+		Sensei_Course_Enrolment::set_store_negative_enrolment_results( false );
```

Comment:
I think that we expose internal functionality here. Maybe adding an optional argument in `recalculate_enrolments` and `is_enrolled` is better?

Reply:
I considered that, but then thought it was a strange argument to pass along. Let me see...